runbotics/runbotics-orchestrator-ui/src/src-app/components/InfoPanel/ProcessInstanceEventsDetails/ProcessInstanceEventsDetails.tsx

```diff
@@ -40,6 +42,14 @@ const ProcessInstanceEventsDetails: VFC<ProcessInstanceEventsDetailsProps> = ({
         all: { events, eventsBreadcrumbTrail, nestedEvents: loopEvents },
     } = useSelector(processInstanceEventSelector);
     const { active } = useSelector(processInstanceSelector);
+    const { draft: { process } } = useSelector(processSelector);
+    const processId = process?.id;
+    const isProcessQueuedOrFailed =
+        processId &&
+        active.jobsMap &&
+        active.jobsMap[processId] &&
+        'eventType' in active.jobsMap[processId] &&
+        JOB_STATUSES.includes(active.jobsMap[processId]?.eventType);
```

Comment:
looks like you're already using this in ProcessInstanceDetails -> let's move it to utils